action.yml

```diff
@@ -0,0 +1,36 @@
+name: "Ruff"
+description: "An extremely fast Python linter."
+author: "Ruff Contributor(s)"
+inputs:
+  options:
+    description:
+      "Options passed to Ruff. Use `ruff --help` to see available options. 
+      Default: 'check'"
+    required: false
+    default: "check"
```

Comment:
one more thing, i think `--format github` should be in here in the defaults somewhere

Reply:
Agreed here.  Since this is a GitHub action, I'd go further and suggest isn't an input option with default... but rather that `--format github` is just to be part of the/this github action.

Reply:
used env var.  that looked cleaner [ is in the action.yml file, rather than being in the command/py file ]